newPassword.php

```diff
@@ -0,0 +1,41 @@
+<?php
+session_start();
+$email = $_POST["email"];
+$dob = $_POST["dob"];
+
+include 'connection.php';
+$sql = "SELECT * FROM users WHERE email=$email";
+$result = $connection->query($sql);
+
+if ($row = $result->fetch_assoc) {
+  if ($dob == $row["dob"]) {
+    $_SESSION["forgotPasswordEmail"] = $email;
+    echo "Dob matched. Enter new password";
```

Comment:
Remove `echo` line.

Redirect to `index.php` with an alert `Your password has been reset`.